server/user_account/urls.py

```diff
@@ -15,5 +15,6 @@
     path('', include(router.urls)),
     path('login/', views.LoginView.as_view()),
     path('logout/', views.LogoutView.as_view()),
-    path('getSomeClients/', views.AccessSomeClients.as_view())
+    path('getSomeClients/', views.AccessSomeClients.as_view()),
+    path('modifyClients/', views.ModifyClient.as_view()),
```

Comment:
Minor but I would rename this endpoint to modifyClient/ instead of modifyClients/ because you are modifying one client at a time.